test/common/upstream/health_checker_impl_test.cc

```diff
@@ -531,6 +531,9 @@ TEST_F(HttpHealthCheckerImplTest, SuccessServiceCheck) {
         EXPECT_NE(nullptr, headers.Path());
         EXPECT_EQ(headers.Host()->value().c_str(), host);
         EXPECT_EQ(headers.Path()->value().c_str(), path);
+        EXPECT_TRUE(headers.Scheme());
```

Comment:
nit: this check and the one on the next line are redundant with the third check. I would remove them. (Similar elsewhere if you copied this.)

Reply:
Agree! I kept to what I found in the file.

Reply:
Done.